server/modlog.ts

```diff
@@ -498,8 +511,13 @@ export class Modlog {
 
 			args.push(...Array(6).fill(search.anyField));
 
-			query += ` OR regex(?, note)`;
-			args.push(this.generateIDRegex(toID(search.anyField)));
+			if (this.useFTS) {
+				query += ` OR note MATCH ?`;
```

Comment:
No way that this works, you need to `JOIN modlog_fts ON modlog_fts.rowid = modlog_id` and match on `modlog_fts.note`